### DISCUSSION TITLE
Using with Ember Framework

### DISCUSSION THREAD

**Discussion by Jazzwal4980** [Drarig29/brackets-viewer.js#110] [Q&A] [answered]
How do I implement this project with Ember framework as Ember does not utilize `<script>` tags? Is there a specific place to import these libraries? (unless I am understanding incorrectly).

**Comment by Drarig29** [comment, marked as answer]
What you want to do isn't specific to this project, you can try to look at StackOverflow: https://stackoverflow.com/a/57813081/3970387

I don't know Ember, so I can't help more, sorry.

**Reply by Jazzwal4980** [reply]
Oh okay, thank you for the response :) Yea I've been surfing the overflow, but I figured I'd ask here to see if I could get it straight from the horse's mouth, thanks for the post reference as well :)
